client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,1316 @@
+								) }
+							/>
+						</React.Fragment>
+					) ) }
+
+				{ featureComparison.map( ( item, index, arr ) => (
+					<React.Fragment key={ `feature${ index }` }>
+						<FeatureTitle
+							key={ `featureTitle${ index }` }
+							className={ `tabbed-plans__feature-title-${ index + 1 }` }
+							isLast={ arr.length === index + 1 }
+						>
+							{ item.featureName }
+							<InfoPopover
+								className="tabbed-plans__feature-tip-info"
+								position="right"
+								icon="info"
+								iconSize={ 15 }
+							>
+								{ item.tooltip }
+							</InfoPopover>
+						</FeatureTitle>
+						{ item.planOne && (
+							<>
+								<Feature
+									key={ `featureItemOne${ index }` }
+									className={ `tabbed-plans__feature-1-${ index + 1 }` }
+									included={ item.planOne[ termLength ].included }
+									isLast={ arr.length === index + 1 }
+								>
+									<span>{ item.planOne[ termLength ].copy }</span>
+									{ item.planOne[ termLength ].mobileCopy && (
+										<span>{ item.planOne[ termLength ].mobileCopy }</span>
+									) }
+								</Feature>
+							</>
+						) }
+						{ item.planTwo && (
+							<Feature
+								key={ `featureItemTwo${ index }` }
+								className={ `tabbed-plans__feature-2-${ index + 1 }` }
+								included={ item.planTwo[ termLength ].included }
+								isLast={ arr.length === index + 1 }
+							>
+								<span>{ item.planTwo[ termLength ].copy }</span>
+								{ item.planTwo[ termLength ].mobileCopy && (
+									<span>{ item.planTwo[ termLength ].mobileCopy }</span>
+								) }
+							</Feature>
+						) }
+						{ item.planThree && (
+							<Feature
+								key={ `featureItemThree${ index }` }
+								className={ `tabbed-plans__feature-3-${ index + 1 }` }
+								included={ item.planThree[ termLength ].included }
+								isLast={ arr.length === index + 1 }
+							>
+								<span>{ item.planThree[ termLength ].copy }</span>
+								{ item.planThree[ termLength ].mobileCopy && (
+									<span>{ item.planThree[ termLength ].mobileCopy }</span>
+								) }
+							</Feature>
+						) }
+					</React.Fragment>
+				) ) }
+				<FreeBanner>
+					{ selectedTab === 'Professional' && (
+						<>
+							Need something simple to start?{ ' ' }
+							<button href="#" onClick={ () => toggleTab() }>
+								Explore our Starter plans
+							</button>
+						</>
+					) }
+					{ selectedTab === 'Starter' && (
+						<>
+							Need eCommerce or professional features?{ ' ' }
+							<button href="#" onClick={ () => toggleTab() }>
+								Explore our Professional plans
+							</button>
+						</>
+					) }
+				</FreeBanner>
+				<SharedFeatures selectedTab={ selectedTab } sharedFeatures={ sharedFeatures } />
+			</Grid>
+		</>
+	);
+}
+
+TabbedPlans.propTypes = {
+	onUpgradeClick: PropTypes.func,
+};
+
+const mapStateToProps = ( state, ownProps ) => {
+	const { plans } = ownProps;
+	// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
```

Comment:
Nice. I'm wondering if it's even possible to make this linting rule smarter, as this is definitely not a bound selector 🤔